packages/firestore/src/remote/datastore.ts

```diff
@@ -87,7 +87,10 @@ export class Datastore {
       'Commit',
       params
     ).then(response => {
-      return this.serializer.fromWriteResults(response.writeResults);
+      return this.serializer.fromWriteResults(
+        response.writeResults,
+        response.commitTime
```

Comment:
Does this need a bang for strict null checks? (i.e. `response.commitTime!`)

I ask because the assignment to `const commitVersion` uses one below.